.appveyor.yml

```diff
@@ -132,3 +135,83 @@ test_script:
 
   - ps: Write-Host "[+] Starting 'npm run test' ..."
   - bash -e -l -c "cd $APPVEYOR_BUILD_FOLDER && export PYINSTALLER=1 && npm run test"
+
+  - ps: Write-Host "[+] Preparing 'native_main.exe' for upload ..."
```

Comment:
Hi @bovine3dom 

Off the top of my head, I think the code section is here because of
the following:

  1. In AppVeyor build pipeline, `test_script` section is executed
     _after_ the `build_script` section

  2. Considering [1], I wanted to make sure that both build and test
     succeeded before I proceeded to grab the artifact(s) for upload.

Hope this helps. Thanks.

Reply:
Ok, thanks.

I'll try to get this merged this evening.